repository/src/main/java/org/alfresco/repo/content/transform/LocalTransformServiceRegistry.java

```diff
@@ -187,20 +182,8 @@ public void register(Transformer transformer, Map<String, Set<TransformOption>>
                         strictMimeTypeCheck, strictMimetypeExceptions, retryTransformOnDifferentMimeType,
                         transformsTransformOptions, this);
             }
-            else if (isPipeline && isFailover)
-            {
-                throw new IllegalArgumentException("Local transformer " + name +
-                        " cannot have pipeline and failover sections. Read from "+readFrom);
-            }
             else if (!isPipeline && !isFailover)
             {
-                baseUrl = getBaseUrlIfTesting(name, baseUrl);
-                if (baseUrl == null)
-                {
-                    throw new IllegalArgumentException("Local transformer " + name +
-                            " must have its baseUrl set in " + LOCAL_TRANSFORMER+name+URL+
-                            " Read from "+readFrom);
-                }
                 int startupRetryPeriodSeconds = getStartupRetryPeriodSeconds(name);
```

Comment:
The 3 removed error check sections above have been moved to CombinedConfig removeOverriddenOrInvalidTransformers which is run before this method.